examples/prometheus-alerting-rules/alerts.yaml

```diff
@@ -0,0 +1,25 @@
+groups:
+- name: kube-state-metrics
+  rules:
+  - alert: KubeStateMetricsListErrors
+    annotations:
+      message: kube-state-metrics is experiencing list errors
+    expr: |
+      (sum(rate(kube_state_metrics_list_total{job="kube-state-metrics",result="error"}[5m]))
+        /
+      sum(rate(kube_state_metrics_list_total{job="kube-state-metrics"}[5m])))
+      > 0.01
+    for: 10m
+    labels:
+      severity: critical
```

Comment:
I think this is critical, but open to discussion. :)

Reply:
There are various alerting philosophies around this, my personal opinion is that when something is not able to be monitored it’s critical, so I think we’re good with having this here.

Reply:
On a related note we should probably add to the message of the alerts that this likely causes kube state metrics not to expose any metrics when it’s scraped so no monitoring of kube objects is working.

Reply:
@brancz  Any wording suggestion on that?

Reply:
`kube-state-metrics is experiencing elevated errors in list operations. This is likely causing it to not be able to expose metrics about Kubernetes objects correctly.`

Reply:
small nit. I would say something like `experiencing errors at an elevated rate.`